docs/src/rules/no-div-regex.md

```diff
@@ -8,15 +8,15 @@ related_rules:
 
 
 
-Require regex literals to escape division operators.
+Require regex literals to escape equal signs at the beginning.
```

Comment:
Our convention for rule docs is that opening sections describe the problem, not what the rule enforces, so while we're here, we could change this to something like:

```suggestion
Characters `/=` at the beginning of a regular expression literal can be confused with a division assignment operator.
```